subsys/bluetooth/mesh/crypto_tc.c

```diff
@@ -23,6 +24,13 @@
 
 #include "mesh.h"
 #include "crypto.h"
+#include "prov.h"
+
+static struct {
```

Comment:
It will help save some memory if `key` is moved to some provisioning context that could be allocated on heap/stack.

Reply:
Can you add these planned improvements to some task?